crates/client/analytics/src/lib.rs

```diff
@@ -196,6 +196,24 @@ pub fn register_counter_metric_instrument<T: CounterType<T> + Display>(
     T::register_counter(crate_meter, instrument_name, desc, unit)
 }
 
+pub trait UpDownCounterType<T> {
+    fn register_counter(meter: &Meter, name: String, description: String, unit: String) -> UpDownCounter<T>;
```

Comment:
I would rename that `register_up_down_counter` to align with other declarations

Reply:
NA